src/anm/graphics/path.js

```diff
@@ -62,6 +63,10 @@ function Path(val) {
         this.segs = val;
     }
 
+    if (engine.Path2D) {
+        this.path2d = new Path2D(Path.toSVGString(this));
```

Comment:
we already have this path parsed, so why convert it to svg string again? i believe it will be faster (and less memory) to create the path using the Path2D api.